DigitalLearningSolutions.Web/Views/NotificationPreferences/_NotificationPreferenceList.cshtml

```diff
@@ -1,24 +1,40 @@
-﻿@using DigitalLearningSolutions.Web.ViewModels.MyAccount
+﻿@using DigitalLearningSolutions.Web.Helpers

```

Comment:
Should we rename this to _NotificationPreferencesExpander now?

Reply:
I prefer to avoid renaming things once merged to master but it makes sense, I'll do this. Not like there's much history to it.